### ISSUE TITLE
Border Control Still Unreliable

### ISSUE THREAD

**Issue by lankaukk** [concrete-utopia/utopia#5831]
The border style section in the inspector was recently fixed, and works correctly on some projects but not in others. cc @liady since you were the one working on this, feel free to tag someone else

![Image](https://github.com/concrete-utopia/utopia/assets/47405698/acfb0f1f-4285-4fc8-915c-2ca2959b886b)

**Comment by liady** [comment]
yes, since the border actually has a color different than black, so we don't recognize it as a "non-existing" border.

![Image](https://github.com/concrete-utopia/utopia/assets/7003853/ffa842cd-a70e-4d4f-9008-c599cfbd3cdd)
![Image](https://github.com/concrete-utopia/utopia/assets/7003853/d98d850c-8ed3-4cc9-913c-f081eb75ac80)

the question is - do we want to treat any 0px border as a "non existent"? This can cause a situation where a user will change the border color (but will leave it at 0px width) - and after reselecting the element they will not see the controller again (since we'll treat it as "no border")
